tests/test_sqs/test_sqs.py

```diff
@@ -199,7 +234,7 @@ def test_send_message_with_attributes():
     message = queue.new_message(body)
     message_attributes = {
         'test.attribute_name': {'data_type': 'String', 'string_value': 'attribute value'},
-        'test.binary_attribute': {'data_type': 'Binary', 'binary_value': 'binary value'},
+        'test.binary_attribute': {'data_type': 'Binary', 'binary_value': base64.b64encode('binary value')},
```

Comment:
@JackDanger For Python 3.x compatibility, I think this should be `base64.b64encode(b'binary value')` - see failing test on travis-ci: https://travis-ci.org/spulec/moto/jobs/231174489#L1635